templates/emails/collective.apply.for.host.hbs

```diff
@@ -98,7 +98,7 @@ Dear admin of {{host.name}},
 <center>
   <a
     class="btn blue"
-    href="{{config.host.website}}/{{host.slug}}/admin/pending-applications#application-{{collective.id}}"
+    href="{{config.host.website}}/{{host.slug}}/admin/host-applications?accountSlug={{collective.slug}}"
```

Comment:
Not a fan of using the `slug` as collective admins can change it and the link would become invalid while the collective still exists and still has an application pending.

Reply:
Ok, let's send the id then.

Reply:
Good point, fixed this to use the id instead